### PR TITLE
Added AssistantAI plugin

### PR DESCRIPTION
**Pull request by kanutron**
This is AssistantAI plugin for Sublime Text:
https://github.com/kanutron/AssistantAI

Sublime Text plugin that allows the user to query API end points and perform text operations with their responses.

- [x] I'm the package's author and/or maintainer.
- [x] I have have read [the docs][1].
- [x] I have tagged a release with a [semver][2] version number.
- [x] My package repo has a description and a README describing what it's for and how to use it.
- [x] My package doesn't add context menu entries. *
- [x] My package doesn't add key bindings. **
- [x] Any commands are available via the command palette.
- [x] Preferences and keybindings (if any) are listed in the menu and the command palette, and open in split view.
- [x] If my package is a syntax it doesn't also add a color scheme. ***
- [x] I use [.gitattributes][3] to exclude files from the package: images, test files, sublime-project/workspace.

My package is a plugin in Python 3.8, to allow other plugins to make API requests to endpoints, as ChatGPT.

There are no packages like it in Package Control.

My package is similar to [OpenAI-sublime-text](https://github.com/yaroslavyaroslav/OpenAI-sublime-text) However it should still be added because it's a more general approach and extensible with other specialized plugins. I expect this package be widely used to provide API endpoints. I'm intended to upload two more packages consuming this AssistantAI package as the base implementation. 

[1]: https://packagecontrol.io/docs/submitting_a_package
[2]: https://semver.org
[3]: https://www.git-scm.com/docs/gitattributes#_export_ignore

### CODE REVIEW

**Comment by kanutron** [comment]
Now it's ready and much cleared.

I did the three changes.

@packagecontrol-bot please proceed.

**Comment by kanutron** [comment]
@packagecontrol-bot please

**Comment by braver** [comment]
Alright, good looking package and different enough in approach that it warrants adding it. As with the OpenAI package, please clearly place a note in the README about how or when potentially code will be sent to third parties. This is relevant because users might not be aware, and might work on code where that is not safe/allowed. See https://github.com/yaroslavyaroslav/OpenAI-sublime-text#note

File: repository/a.json [hunk 1]
```diff
@@ -1974,6 +1974,18 @@
 				}
 			]
 		},
+		{
+			"name": "AssistantAI",
+			"details": "https://github.com/kanutron/AssistantAI",
+			"author": "kanutron",
```
Comment (braver) [inline repository/a.json:1980]: This is simply your username, so you can omit it.

Comment (kanutron) [reply repository/a.json]: OK. Removed the 'author' key.

**Comment by kanutron** [comment]
> Alright, good looking package and different enough in approach that it warrants adding it.

Thanks for the feedback and the review.

> As with the OpenAI package, please clearly place a note in the README about how or when potentially code will be sent to third parties. This is relevant because users might not be aware, and might work on code where that is not safe/allowed. See https://github.com/yaroslavyaroslav/OpenAI-sublime-text#note

Indeed, this is an important issue to remark that I missed. I added in the README two notes:
- A "Privacy warning" at the top of the README
- A note about "paid service" in the OpenAI section

I hope this is helps.

**Comment by braver** [comment]
Thanks!

**Pull request merged**